sauce/features/general/better-scrollbars/settings.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  name: 'BetterScrollbars',
+  type: 'select',
+  default: 'small',
```

Comment:
Shouldn't the default be a number that corresponds to the value for Default? i.e. '0'